tests/integration/helpers/upgrade.py

```diff
@@ -113,3 +114,46 @@ async def set_fcv(
         ops_test, app_name, substrate, replica_set_uri, admin_mongod_cmd, expecting_output=False
     )
     assert result.succeeded, f"Failed to set fcv to {fcv}."
+
+
+async def set_password_action(
```

Comment:
do we still need this?

Reply:
But this is for MongoDB 8, we don't have upgrades yet - the first revision upgrades will be based on will not have those actions